lib/workers/repository/errors-warnings.ts

```diff
@@ -83,9 +87,13 @@ export function getDepWarningsOnboardingPR(
 
 export function getDepWarningsPR(
   packageFiles: Record<string, PackageFile[]>,
+  config: RenovateConfig,
   dependencyDashboard?: boolean
 ): string {
   const { warnings, warningFiles } = getDepWarnings(packageFiles);
+  if (config.suppressNotifications?.includes('dependencyLookupWarnings')) {
```

Comment:
🤔 this suppresses all warnings. i don't think this is right. we probably need to filter them instead of return early.

@rarkins WDYT?

Reply:
Yes you're right

Reply:
Ok I'll take a look at this first thing monday.

Reply:
Are we suggesting that in the instance of the onboardingPR we need to filter but the other ones are ok?

Reply:
So I've had a look at the code and Im not 100% sure what else we would do in this situation.
The only warnings in that block are dependency lookup warnings which is what the suppression is relating to.
So dropping out when we do, won't stop anything else been logged out as its all related to dep lookup warnings?

Reply:
Unless I'm reading it wrong? 

The logger.debug calls them package lookup warnings but they appear to be the same thing as dep lookup warnings as the warning text outputs them under this.

```
  warningText = emojify(
    `\n---\n\n### :warning: Dependency Lookup Warnings :warning:\n\n`
  );
```

Reply:
I think you're right - it's only dep lookup warnings. In that case ok to ignore all

Reply:
@viceice Happy to accept the review now?